LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2020 Thomas Bui
+Copyright (c) 2022 Devicons
```

Comment:
Not sure if we are allowed to change this. We at the very least have to make sure with @Thomas-Boi I think

Reply:
Right. I'll just keep the year changed to 2022. Sounds good?

Reply:
No, the year should be the year the copyright started, so that shouldn't be changed either

Reply:
Alright. And not even `2020-2021`? The latest changes by Thomas were on that year.

Reply:
A date range is not needed, see [this stackexchange thread](https://opensource.stackexchange.com/questions/1522/what-should-be-written-in-mit-license-year-full-name) for more info.

More info about changing licenses: https://opensource.stackexchange.com/questions/8853/can-i-change-the-copyright-holder-name-and-year-of-an-mit-license

Reply:
Great research, thanks! :)
We should only change if we have permission from the copyright holder, in this case, Thomas Bui. But still, there's no harm in keeping this as-is.

Reply:
@Snailedlt and @Panquesito7 feel free to change the copyright to Devicon. The only thing that I request is that the code stays MIT license (so others can use it) and that I can use it in other places without any restriction 😃